app/isomorphic/components/layouts/footer/index.js

```diff
@@ -1,49 +1,31 @@
 import React from "react";
-import PT from "prop-types";
-import { connect } from "react-redux";
-import get from "lodash/get";
-import { Link } from "@quintype/components";
 import "./styles.m.css";
+import logo from "./logo.png";
```

Comment:
check other files for spacing, this is for readability

Reply:
Yeah have changed it